test/ruby_lsp_rails/server_test.rb

```diff
@@ -0,0 +1,16 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+require "ruby_lsp/ruby_lsp_rails/server"
+
+class ServerTest < ActiveSupport::TestCase
```

Comment:
@vinistock since the server is in a different process, this needs to be tested differently from the other aspects in test/ruby_lsp_rails/runner_client_test.rb. But I think it would make sense to move some of the others test to here too, and leave just model returns information for the requested model" as an integration test?

Reply:
Pushed a commit to show how that would look.